test/e2e/specs/specs-i18n/wp-i18n__logged-out-redirect.ts

```diff
@@ -2,19 +2,17 @@
  * @group i18n
  */
 
-import { setupHooks, DataHelper, BrowserHelper } from '@automattic/calypso-e2e';
+import { setupHooks, DataHelper, TestEnvironment } from '@automattic/calypso-e2e';
 import { Page } from 'playwright';
 
-const locale = BrowserHelper.getLocale();
-
-describe( `Logged out homepage redirect test @i18n (${ locale })`, function () {
+describe.skip( 'I18N: Homepage Redirect', function () {
```

Comment:
I've skipped this test for now because this test doesn't actually do anything.

What I observe is as follows:
1. browser is launched
2. page navigates to `https://wordpress.com`

Then nothing occurs after that, until the next call in Line 20 times out since nothing happened.

@yuliyan, is there something missing from this portion? I am not sure how the original Selenium spec looked like, so perhaps it's missing a portion like setting the locale.

Reply:
> It's testing the locale redirects for https://wordpress.com, that are based on the browser language settings that set the Accept-Language header for request. For example, if you set Spanish as your primary browser language and you go to `https://wordpress.com` (logged-out), you should be redirected to `https://wordpress.com/es`. For Playwright, we specify [the `locale` config when creating new browser context](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/browser-helper.ts#L89) in order to set the browser language.

Got it, thanks. I see why this test does not seem to do 'anything' at this moment - the BrowserContext is created with `en` as the locale and this persists for the duration of the test and this is why the redirect expectation fails.

I have some ideas about how to work around this, by implementing a new method for BrowserManager to help set the locale. I'll give that a try and if it works incorporate them to the PR.